opentelemetry-api/src/opentelemetry/metrics/__init__.py

```diff
@@ -40,7 +40,7 @@
     Union,
 )
 
-from opentelemetry.util import _load_meter_provider
+from opentelemetry.util.providers import _load_meter_provider
```

Comment:
should the setup.cfg also be updated to use `find_namespace_package`?

https://github.com/open-telemetry/opentelemetry-python/blob/main/opentelemetry-api/setup.cfg#L41

The need to use find_namespace_packages is in the [docs you linked](https://packaging.python.org/guides/packaging-namespace-packages/#native-namespace-packages)

Reply:
I really don't know. I don't find it clear if this (as used in `setup.cfg`):
```
[options]
...
packages=find_namespace:
```
is equivalent to `setuptools.find_namespace_packages()`.

If found [this](https://setuptools.readthedocs.io/en/latest/userguide/package_discovery.html?highlight=find_namespace#using-find-namespace-or-find-namespace-packages). I _think_ the section title implies that `find_namespace:` in `setup.cfg` is equivalent to `find_namespace_packages` (the [previous section](https://setuptools.readthedocs.io/en/latest/userguide/package_discovery.html?highlight=find_namespace#using-find-or-find-packages) seems to imply that `find:` in `setup.cfg` is equivalent to `find_packages`) but I am not completely sure.

Anyways, the code seems to be working with `find_namespace:` in `setup.cfg` so I'll leave it as it is now. :shrug:

Reply:
thanks! Yeah I think the section is clear on that.